parcel_test.go

```diff
@@ -31,75 +31,108 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
-	store := NewParcelStore(db)
-	parcel := getTestParcel()
+	db, err := sql.Open("sqlite", "tracker.db") // подключение к БД
+	require.NoError(t, err)                     // проверка ошибки
+	store := NewParcelStore(db)                 // создание тестовой посылки
+	parcel := getTestParcel()                   // получение тестовой посылки
+	defer db.Close()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	parcel.Number, err = store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
+	checkParcel, err := store.Get(parcel.Number)
+	require.NoError(t, err)
+	require.Equal(t, parcel, checkParcel)
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	err = store.Delete(parcel.Number)
+	require.NoError(t, err)
+
 	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(parcel.Number)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для проверки ошибок лучше использовать специальные функции
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```